app/root.tsx

```diff
@@ -253,7 +253,7 @@ export function Layout({ children }: { children?: ReactNode }) {
         <DevBanner />
         <Header />
         <NewsBanner>
-          New! Circulars over Kafka, Heartbeat Topic, and Schema v4.1.0. See{' '}
+          New! October 18 GCN Classic Outage and Schema v4.2.0. See{' '}
```

Comment:
The text "New! ... Outage" seems pretty weird.

Reply:
Ok, what if we turn it around?  
New! Schema v4.2.0 and October 18 GCN Classic Outage.